README_de-DE.md

```diff
@@ -0,0 +1,432 @@
+# Commit Messages Guide
+
+Ein Leitfaden um die Wichtigkeit von Commit-Messages zu verstehen und diese richtig zu formulieren.
+
+Verstehe was ein Commit ist, warum es wichtig ist eine gute Commit-Message zu schreiben, Best-Practices und ein paar Tipps um eine saubere Commit-Historie zu planen oder vielleicht sogar nachträglich noch anzupassen.
+
+#### Hinweis des Übersetzers
+
+Auch wenn das eine Übersetzung darstellen soll, empfiehlt es sich, alle Commit-Messages auf Englisch zu schreiben,deshalb sind die Beispiele nicht übersetzt.
```

Comment:
space after comma: `schreiben, deshalb`

Reply:
good catch, will fix

Reply:
Please add the space, so that we can :ship: it! :wink:

Reply:
just did. happy 🚢-ing